c_src/membrane_agora_plugin/noop.cpp

```diff
@@ -0,0 +1,18 @@
+/*
+ This is a small noop file so bundlex has something to compile on non-linux machines.
+ The code below is the smallest nif loading code (from the erlang docs) since typically
+ unifex will add it, however, we don't want to include unifex here.
+ */
+#include <erl_nif.h>
+
+static ERL_NIF_TERM hello(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
+{
+    return enif_make_string(env, "Hello world!", ERL_NIF_LATIN1);
+}
+
+static ErlNifFunc nif_funcs[] =
+{
+    {"hello", 0, hello}
+};
+
+ERL_NIF_INIT(niftest,nif_funcs,NULL,NULL,NULL,NULL)
```

Comment:
As discussed in the private conversation - the first argument of the `ERL_NIF_INIT` macro defines the Erlang module name that is capable of loading that NIF. We need to define separate NIFs for `Elixir.Membrane.Agora.Sink.Native.Nif` and for `Elixir.Membrane.Agora.Source.Native.Nif`